docs/server-side/v2/api-reference/analytics/overview.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Overview
+nav: 3.61
+---
+
+## Analytics APIs
+
+Analytics APIs are used to fetch events with an HTTP request. Currently, we support retrieving [track events](/server-side/v2/api-reference/analytics/track-events).
+
+Event data can be queried up to last 30 days.
+
+### What can I build?
+
+-   You can use track events to get a better understanding of user activity and build tools around it as explained in [use cases](/server-side/v2/api-reference/analytics/track-events#why-would-you-use-this-api)
```

Comment:
i would just move the use-cases section from that page to here

Reply:
The problem I find with this is that I am using track-specific details e.g `mute` , and `source`  in the logic/pseudocode. When a user is landing on the overview page these details might feel overwhelming, wdyt?

Reply:
yeah agreed - my bad, i didn't realise that. let's keep it to track page only.